### ISSUE TITLE
Not enough contrast on the background of the selected lines in menus

### ISSUE THREAD

**Issue by Brakkar** [Microsoft/vscode#7472]
- VSCode Version: 1.2.0

Please see attached image. This is voluntarily a picture to show the context of my low brightness screen.
You can see that the background of the selected line (here "default settings") is not contrasted enough.
I don't think there is a textmate setting to alter this.

Please increase this contrast of bg selected line.

![capture d ecran 2016-06-09 a 14 22 36](https://cloud.githubusercontent.com/assets/5489200/15929564/067586a8-2e4e-11e6-8621-7ecf1e1e5142.jpg)

**Comment by bpasero** [comment]
@Brakkar are you talking about the indication of the currently active line in the editor?

**Comment by Brakkar** [comment]
@bpasero no, the active line in command palette (as shown on attached image) and file explorer.
It should be the same color as the one we set in the editor, or it should have a setting on its own.

Currently, as shown on the photo I attached, it's almost invisible.

**Comment by bpasero** [comment]
Ok, I am not doing anything specific, those are the default tree colors.

**Comment by Brakkar** [comment]
But can't a custom color be implemented for this ? Or at least make it match the editor select line color that we can already set ? tks

**Comment by joaomoreno** [comment]
But why did you configure your screen to have such a low contrast? This seems like a problem with your screen settings instead of a problem with Code itself. I'm sure you'll find many other applications having this issue, once you bring down your screen's contrast so much.

**Comment by bpasero** [comment]
With the theming work I think you should check for another theme with more contrast or adjust the selection color via settings.
